tests/python/unittest/test_cp_async_in_if_then_else.py

```diff
@@ -0,0 +1,304 @@
+                                        B_reindex_shared_local[v2, v1],
+                                    )
+                                    T.writes(Y_reindex_local[v0, v1])
+                                    T.block_attr(
+                                        {
+                                            "meta_schedule.thread_extent_high_inclusive": 1024,
+                                            "meta_schedule.thread_extent_low_inclusive": 32,
+                                            "meta_schedule.tiling_structure": "SSSRRSRS",
+                                        }
+                                    )
+                                    Y_reindex_local[v0, v1] = (
+                                        Y_reindex_local[v0, v1]
+                                        + A_reindex_shared_local[v2, v0]
+                                        * B_reindex_shared_local[v2, v1]
+                                    )
+                    for ax0, ax1 in T.grid(8, 4):
+                        with T.block("Y_reindex_local"):
+                            T.where(
+                                ax0_0_ax1_0_fused // 8 * 64
+                                + ax0_1_ax1_1_fused // 2 * 32
+                                + ax1_2_0_ax0_2_0_ax0_2_1_ax1_2_1_ax0_2_2_fused // 32 * 16
+                                + ax1_2_0_ax0_2_0_ax0_2_1_ax1_2_1_ax0_2_2_fused % 2 * 8
+                                + ax0
+                                < 1012
+                                and ax0_0_ax1_0_fused % 8 * 128
+                                + ax0_1_ax1_1_fused % 2 * 64
+                                + ax1_2_0_ax0_2_0_ax0_2_1_ax1_2_1_ax0_2_2_fused % 32 // 2 * 4
+                                + ax1
+                                < 1017
+                            )
+                            v0 = T.axis.spatial(
+                                1024,
+                                ax0_0_ax1_0_fused // 8 * 64
+                                + ax0_1_ax1_1_fused // 2 * 32
+                                + ax1_2_0_ax0_2_0_ax0_2_1_ax1_2_1_ax0_2_2_fused // 32 * 16
+                                + ax1_2_0_ax0_2_0_ax0_2_1_ax1_2_1_ax0_2_2_fused % 2 * 8
+                                + ax0,
+                            )
+                            v1 = T.axis.spatial(
+                                1024,
+                                ax0_0_ax1_0_fused % 8 * 128
+                                + ax0_1_ax1_1_fused % 2 * 64
+                                + ax1_2_0_ax0_2_0_ax0_2_1_ax1_2_1_ax0_2_2_fused % 32 // 2 * 4
+                                + ax1,
+                            )
+                            T.reads(Y_reindex_local[v0, v1])
+                            T.writes(Y[v0, v1])
+                            Y[v0, v1] = Y_reindex_local[v0, v1]
+
+
+def test_matmul():
+    with tvm.transform.PassContext(config={"tir.use_async_copy": 1}):
+        rt_mod = tvm.build(Module, target="cuda")
```

Comment:
are async copy instructions always available? particularly on previous generations of cuda gpus

Reply:
now compute version check is added to the unittest